layout/explore/explore-menu/styles.scss

```diff
@@ -4,6 +4,9 @@
     min-width: 200px;
     max-width: 200px;
     height: 100%;
+    position: sticky;
```

Comment:
This won't work in IE, If IE is a requirement you would need to include a polyfill https://github.com/dollarshaveclub/stickybits

Reply:
Thanks for your input!
I didn't know that 😮 
However, we're not prioritizing having IE full support in all the app sections so maybe this is something that could be tackled in the future when necessary. 😉